core/sgf/parser.go

```diff
@@ -395,3 +395,38 @@ func postProcessProperties(n *game.Node, prop string, propData []string) error {
 
 	return nil
 }
+
+// Serialize converts a Game into SGF format.
+// Calls serializeHelper.
```

Comment:
nit: You don't need to mentioned the internals (serializeHelper) in the public docs.